src/sentry/api/serializers/models/organization_member.py

```diff
@@ -64,20 +80,30 @@ def get_attrs(
         external_users_map = defaultdict(list)
 
         if "externalUsers" in self.expand:
-            external_users = list(ExternalUser.objects.filter(organizationmember__in=item_list))
-
-            for external_user in external_users:
-                serialized = serialize(external_user, user)
-                external_users_map[external_user.organizationmember_id].append(serialized)
+            organization_id = get_organization_id(item_list)
+            actor_mapping = {user.actor_id: user for user in users_set}
+            external_actors = list(
+                ExternalActor.objects.filter(
+                    actor_id__in=actor_mapping.keys(),
+                    organization_id=organization_id,
+                )
+            )
+
+            for external_actor in external_actors:
+                serialized = serialize(external_actor, user, key="user")
```

Comment:
I'd recommend you serialize the entire list of `external_actors` in bulk, then build this mapping based off of that. That way any necessary bulk querying can happen